third-party/devtools-tests/e2e/lighthouse/navigation_test.ts

```diff
@@ -111,7 +112,7 @@ describe('Navigation', async function() {
           assert.strictEqual(numNavigations, 6);
         }
 
-        assert.strictEqual(lhr.lighthouseVersion, '10.0.1');
+        assert.strictEqual(lhr.lighthouseVersion, packageJson.version);
```

Comment:
I assume the purpose of this is to assert that it's not accidentally testing the wrong version in DT, but let me know what you think of this solution, @adamraine :)

Reply:
This file get's rolled into DT frontend repo so this wouldn't work. We need to hardcode the version here.

Reply:
OK, attempt #2 :)